src/sound/manager.js

```diff
@@ -81,8 +81,15 @@ class SoundManager extends EventHandler {
     }
 
     resume() {
-        this.suspended = false;
-        this.fire('resume');
+        const resumeFunction = function () {
+            this.suspended = false;
+            this.fire('resume');
+        }.bind(this);

```

Comment:
```suggestion
        const resumeFunction = () => {
            this.suspended = false;
            this.fire('resume');
        };
```

Reply:
I'm assuming the above code suggestion works OK. But using the arrow function should avoid the need to use `bind`.

Reply:
Beat ya! :D

Reply:
Yeah, except you deleted the trailing semi-colon, @slimbuck! 😆